gnodeb/worker/gnbamfworker/handler.go

```diff
@@ -450,13 +458,10 @@ func HandleUeCtxReleaseCommand(gnb *gnbctx.GNodeB, amf *gnbctx.GnbAmf,
 
 	if ueNgapIds.Present == ngapType.UENGAPIDsPresentUENGAPIDPair {
 		ranUeNgapId = &ueNgapIds.UENGAPIDPair.RANUENGAPID
-		if ranUeNgapId == nil {
-			amf.Log.Errorln("RANUENGAPID is nil")
-			return
-		}
```

Comment:
What about checking whether the struct is empty instead of deleting this? Because in [line 462](https://github.com/omec-project/gnbsim/blob/main/gnodeb/worker/gnbamfworker/handler.go#L462) we try to get the `Value` from the struct, so it would be good to make sure that the struct is not empty.

Reply:
`ueNgapIds.UENGAPIDPair` is a pointer, so with my latest change now we make sure we do not reference a null pointer.
`RANUENGAPID` and `RANUENGAPID.Value` are not pointers so I cannot check against null. 
is this what you meant?